brackend/endpoints/tournaments/repositories/TournamentRepository.py

```diff
@@ -1,14 +1,45 @@
 from abc import ABC
-from sqlalchemy.orm import Session
+from sqlalchemy.orm import Session, subqueryload
 from brackend.db.models import (
     EngineGetter,
     Tournament,
+    UserTournament,
+    User,
 )
+from brackend.db.enums import UserRole
 
 
 class TournamentRepository(ABC):
     engine = EngineGetter.get_or_create_engine()
 
+    @classmethod
+    def get_by_id(cls, t_id):
+        with Session(cls.engine) as session:
+            tournament = session.query(Tournament) \
+                .options(subqueryload(Tournament.brackets)) \
+                .filter(Tournament.id == t_id) \
+                .one_or_none()
+
+            return tournament
+
+    @classmethod
+    def get_by_id_with_owner(cls, t_id):
+        """
+            Return a tournament with it's owner info
+        """
+        with Session(cls.engine) as session:
+            result = session.query(Tournament, User) \
+                .join(UserTournament, Tournament.id == UserTournament.tournament_id) \
+                .join(User, User.id == UserTournament.user_id) \
+                .filter(Tournament.id == t_id) \
+                .filter(UserTournament.role == UserRole.organizer) \
+                .all()
+
+            tournament = result[0][0]
+            owner = result[0][1]
```

Comment:
I believe there's a way to make this prettier, where we could do something like `result.Tournament` and `result.User` - but this works for now 😓

Reply:
does result.tournament not work for now?

Reply:
Nope 😭 `AttributeError: Could not locate column in row for column 'tournament'`